raspberry-pi/4/pkgs-overlays.nix

```
@@ -2,7 +2,9 @@
 let
   cfg = config.hardware.raspberry-pi."4".apply-overlays-dtmerge;
   dt_ao_overlay = _final: prev: {
-    deviceTree.applyOverlays = prev.callPackage ./apply-overlays-dtmerge.nix { };
+    deviceTree = prev.deviceTree // {
+      applyOverlays = prev.callPackage ./apply-overlays-dtmerge.nix { };
```

nit: It's technically better to use `_final` for `callPackage` just in case another overlay somewhere would change it.

```suggestion
      applyOverlays = _final.callPackage ./apply-overlays-dtmerge.nix { };
```